### PR TITLE
Remove regex bunq/tinker_php#11

### LINKED ISSUE #11: Get new user on (potential) Sandbox reset.

### PR DESCRIPTION
**Pull request by OGKevin**
Closes #11

### CODE REVIEW

**Comment by OGKevin** [comment]
@sandervdo please 👀

File: src/BunqLib.php [hunk 1]
```diff
@@ -90,8 +89,13 @@ public function __construct(BunqEnumApiEnvironmentType $bunqEnumApiEnvironmentTy
...
     /**
      * Restores the context from the saved file during creation.
+     *
+     * @param bool $resetConfigIfNeeded
+     *
+     * @throws BunqException
+     * @throws ForbiddenException
      */
-    private function setupContext()
+    private function setupContext(bool $resetConfigIfNeeded = true)
```
Comment (sandervdo) [inline src/BunqLib.php:98]: No typehint for bool.

**Review by sandervdo — CHANGES_REQUESTED** [review]
@OGKevin yours.

File: src/BunqLib.php (hunk 1 above)
Comment (OGKevin) [reply src/BunqLib.php:98]: This is not the backend tho :P here we follow the psr style guide. AFAIK there is no mention of not type hinting bools 🤔

Comment (sandervdo) [reply src/BunqLib.php:98]: @OGKevin as long as you don't suffer from the weird casting shizzle I'm fine with it.

Comment (OGKevin) [reply src/BunqLib.php:98]: @sandervdo nope we don't suffer from this 😬

Comment (sandervdo) [reply src/BunqLib.php:98]: @OGKevin then fine.

**Review by sandervdo — APPROVED** [review]

**Comment by OGKevin** [comment]
@andrederoos

**Pull request merged**